### DISCUSSION TITLE
甲骨文无法联入

### DISCUSSION THREAD

**Discussion by star2k** [lollipopkit/flutter_server_box#209] [General]
SSHAuthAbortError(Connection closed before authentication)，某vps无法连接成功

**Comment by lollipopkit** [comment]
95%是网络问题，检查代理软件、服务器防火墙设置等...

**Reply by star2k** [reply]
> 95%是网络问题，检查代理软件、服务器防火墙设置等...

之前启用“跳板服务器”就正常，pc上finalshell就可以直接连接，目前没找到原因

**Reply by lollipopkit** [reply]
。。。跳板服务器因为兼容性问题被禁用了，得等上游依赖修复bug

**Reply by star2k** [reply]
但我在电脑端，能直连啊

**Comment by star2k** [comment]
但是我在电脑上可以正常连接啊

SSHAuthFailError(All authentication methods failed)

**Reply by lologu1** [reply]
密钥对了没？

**Reply by star2k** [reply]
> 密钥对了没？

密码和pc端的一样，之前通过内置跳板能连接上

**Reply by hide3110** [reply]
我在连ipv6鸡的时候也是报这个错，是不是ssh不能直连ipv6？
